api/EXT_texture_filter_anisotropic.json

```diff
@@ -6,73 +6,73 @@
         "support": {
           "chrome": [
             {
-              "version_added": true
+              "version_added": "34"
             },
             {
-              "version_added": true,
+              "version_added": "35",
               "prefix": "WEBKIT_"
             }
           ],
           "chrome_android": [
             {
-              "version_added": true
+              "version_added": "34"
             },
             {
-              "version_added": true,
+              "version_added": "35",
               "prefix": "WEBKIT_"
             }
           ],
           "edge": {
-            "version_added": "≤18"
+            "version_added": "12"
```

Comment:
If Chrome still has the prefixed variant, shouldn't Edge also get it? (I half suspect it's not in Chrome any longer.)

Reply:
I forgot to test the prefix in Edge, oops!  It's still around in both Chrome and Edge.

Reply:
Aww, too bad, prefixes are lame.